forge-ai/src/main/java/forge/ai/AiBlockController.java

```diff
@@ -776,15 +772,44 @@ private void reinforceBlockersAgainstTrample(final Combat combat) {
                 continue;
             }
 
+            boolean needsMoreChumpBlockers = true;
+            // See if it's possible to tank up the damage with Banding
+            List<String> bandsWithString = Arrays.asList("Bands with Other Legendary Creatures",
+                    "Bands with Other Creatures named Wolves of the Hunt",
+                    "Bands with Other Dinosaurs");
+            for (final Card blocker : combat.getBlockers(attacker)) {
+                if (blocker.hasKeyword(Keyword.BANDING) || blocker.hasAnyKeyword(bandsWithString)) {
+                    needsMoreChumpBlockers = false;
```

Comment:
This should just lead to `continue` so you avoid calculating `chumpBlockers` and don't need to have the first block if-enclosed

Reply:
Done :) :+1:

Reply:
Done as well, thanks! :)